com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolume.hlsl

```diff
@@ -79,12 +79,31 @@ APVConstants LoadAPVConstants( StructuredBuffer<int> index )
 
 float3 DecodeSH(float l0, float3 l1)
 {
-    return (l1 - 0.5) * 4.0 * l0;
+    // TODO: Enlarging range by 2 because we're using irradiance probes. Need to derive correct factors and remove this.
+    // Compare ProbeGIBaking.cs
+    return (l1 - 0.5f) * 2.0f * l0 * 2.0f;
 }
 
-float4 DecodeSH(float l0, float4 l1)
+void DecodeSH_L2(float l0, inout float4 l2_R, inout float4 l2_G, inout float4 l2_B, inout float4 l2_C)
```

Comment:
I would just adding a comment here about the coefficients

Reply:
Same remark than in C#.   Currently we could define the constant in C# and generate it for the hlsl code, so we are sure they match

Reply:
Done and done